requirements-dev.txt

```diff
@@ -13,15 +13,15 @@ charset-normalizer==3.0.1 ; python_full_version >= '3.6.0'
 click==8.1.3 ; python_version >= '3.7'
 colorama==0.4.6 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6'
 coverage==7.1.0
-cryptography==39.0.0
+cryptography==39.0.1
 distlib==0.3.6
-docutils==0.17.1 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
+docutils==0.18.1 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
 exceptiongroup==1.1.0 ; python_version < '3.11'
 filelock==3.9.0 ; python_version >= '3.7'
 flake8==6.0.0
 gitdb==4.0.10 ; python_version >= '3.7'
-gitpython==3.1.30 ; python_version >= '3.7'
-identify==2.5.17 ; python_version >= '3.7'
+gitpython==3.1.31 ; python_version >= '3.7'
```

Comment:
Where do we use that?

Reply:
It is a dynamic dependency from our listed one in Pipfile development section.